### PR TITLE
OXT-1525: linux-firmware: update to latest rev

### PR DESCRIPTION
**Pull request by jean-edouard**
Signed-off-by: Jed <lejosnej@ainfosec.com>

### CODE REVIEW

**Comment by stacktrust** [comment]
Please create a JIRA ticket to track this, so we can increase the chance of it being scheduled for future updates.

**Comment by jean-edouard** [comment]
https://openxt.atlassian.net/browse/OXT-1525

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6461

File: recipes-kernel/linux-firmware/linux-firmware_git.bb [hunk 1]
```diff
@@ -138,6 +138,9 @@ S = "${WORKDIR}/git"
 
 inherit allarch update-alternatives
 
+# OE started complaining about the architecture of the binaries for some reason
+INSANE_SKIP = "arch"
```
Comment (eric-ch) [inline recipes-kernel/linux-firmware/linux-firmware_git.bb]: [upstream did too](https://github.com/openembedded/openembedded-core/commit/292eef00b889c61d91c3c965fd3a1009a85f7aa7)

Apparently some recent firmwares started triggering the QA check.

**Comment by eric-ch** [comment]
Will merge soon.

**Pull request merged**